src/main/java/Circle.java

```diff
@@ -0,0 +1,18 @@
+public class Circle extends GeometricObject {
+
```

Comment:
let's have here field radius that will be initialized through constructor, and based on that we will calculate area

Reply:
done